bliss/utils.py

```diff
@@ -0,0 +1,41 @@
+import numpy as np
+import torch
+from torch import Tensor
+
+from bliss.catalog import FullCatalog
+
+
+def convert_mag_to_flux(mag: Tensor, nelec_per_nmgy=987.31) -> Tensor:
```

Comment:
If we're using this function in non-sdss-specific contexts, we may want to call it something like `convert_mag_to_nmgy` instead. Because a nanomaggy is a specific unit of flux.

Also, I'm not sure if `987.31` is the right constant to use outside of sdss. (it could be)

Reply:
Reverted the commit that moved this to utils - now the function is back in `sdss.py`, but it's currently not used by any code. `DecalsFullCatalog` has its own version of the function without the `nelec_per_nmgy` argument - not sure what the units are meant to be, but this is used when loading the Decals catalog:
https://github.com/prob-ml/bliss/blob/2f16bf3074aa4475b899f4292768c18afd7cafe6/bliss/surveys/decals.py#L22-L24